x-pack/plugins/transform/public/app/common/validators/is_continuous_mode_delay.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Validates continuous mode time delay input.
+ * Doesn't allow floating intervals.
+ * @param value User input value.
+ */
+export function isContinuousModeDelay(value: string): boolean {
```

Comment:
we have a generic [patternValidator](https://github.com/elastic/kibana/blob/8453fe820a7bd0b264b3963f547a5092b5e811d5/x-pack/plugins/ml/common/util/validators.ts#L33)

Reply:
Looks like `patternValidator` lives in the `ml` plugin, not `transform`. We could at one point create a package for validators and share that among plugins, but I'm afraid it's not something I'm going to pick up in this PR 😅 .